tools/generate-helpers/pg-table-bindings/list.go

```diff
@@ -68,6 +68,7 @@ func init() {
 		&storage.NodeComponentEdge{}:                            resources.Node,
 		&storage.NodeCVE{}:                                      resources.Node,
 		&storage.NotificationSchedule{}:                         resources.Notifications,
+		&storage.Notification{}:                                 resources.Administration,
```

Comment:
First off, these two lines are confusing.

How related are NotificationSchedule and Notifications and why do they have a different SAC resource? I'm not quite sure what the purpose of the `Notifications` internal SAC resource is—you might want to check with @rhybrillou—but FWIW I think API token expirations should be sent to the notification service you work on.

Reply:
They are not related. I guess this highlights your previous point about the naming.

> but FWIW I think API token expirations should be sent to the notification service you work on

Sorry I don't understand, what do you mean by this?

Reply:
It looks like `NotificationSchedule` has been recently introduced by Yann to support sending reminders for about to expire API tokens. I'm not quite sure why a SAC resource is needed there but his use case seems to be a subset of the notifications work you do.

Reply:
Clarified this offline as well, the `NotificationSchedule` one is different from this (and since the naming changed from `Notification` to `AdministrationEvent` the ambiguity has also been somewhat resolved).

It'll be addressed separately.